bundles/org.openhab.binding.surepetcare/src/main/java/org/openhab/binding/surepetcare/internal/utils/GsonLocalDateTypeAdapter.java

```diff
@@ -0,0 +1,71 @@
+package org.openhab.binding.surepetcare.internal.utils;
```

Comment:
The copyright header is missing for this file.

Reply:
Done